config/kernel/linux-odroidc1-next.config

```diff
@@ -93,7 +93,9 @@ CONFIG_TICK_CPU_ACCOUNTING=y
 # CONFIG_VIRT_CPU_ACCOUNTING_GEN is not set
 # CONFIG_IRQ_TIME_ACCOUNTING is not set
 # CONFIG_BSD_PROCESS_ACCT is not set
-# CONFIG_TASKSTATS is not set
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
```

Comment:
This is not getting enabled because https://cateee.net/lkddb/web-lkddb/TASK_IO_ACCOUNTING.html depends on depends on: CONFIG_TASK_XACCT

Its not a copy paste operating which is why we ask you to submit a (working and tested) configuration which is:

```diff --git a/config/kernel/linux-odroidc1-next.config b/config/kernel/linux-odroidc1-next.config
index de56766d..e8eec9a7 100644
--- a/config/kernel/linux-odroidc1-next.config
+++ b/config/kernel/linux-odroidc1-next.config
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm 5.2.1 Kernel Configuration
+# Linux/arm 5.2.2 Kernel Configuration
 #
 
 #
@@ -93,7 +93,10 @@ CONFIG_TICK_CPU_ACCOUNTING=y
 # CONFIG_VIRT_CPU_ACCOUNTING_GEN is not set
 # CONFIG_IRQ_TIME_ACCOUNTING is not set
 # CONFIG_BSD_PROCESS_ACCT is not set
-# CONFIG_TASKSTATS is not set
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_XACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
 # CONFIG_PSI is not set
 # end of CPU/Task time and stats accounting
```

Reply:
Ok, sorry.